.github/workflows/on-pr-and-trunk_one-app-unit-and-lint-tests.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x, "^18 < 18.16"]
+        node-version: [16.x, 18.x]
```

from https://github.com/americanexpress/one-app/actions/runs/6042820335/job/16398714236?pr=1090
perhaps this should instead be
```suggestion
        node-version: [18.x]
```
until 20.x is LTS?

---

maybe revert the engines field. This still supports 16 for another week or so

---

whoops, forgot that change was me, good catch!

---

reverted the engines and tests of it in c63a856